travis.sh

```diff
@@ -45,12 +45,12 @@ then
       rbenv install --verbose $ruby_version
     )
 
-    gem pristine --all
-    gem update --no-document --system
-    gem update --no-document
+    yes | gem pristine --all
```

Comment:
I didn't understand why it is required

Reply:
In some builds, while these `gem` commands are being executed, it asks "Overwrite the executable? [yN]" and it waits for `yes` or `no` input and after 10 mins travis times out. In order to automatically answer `yes` if such case occurs, `yes | ` is prepended.

Reply:
Yeah! That I understood. But I believe without also it is fine. Anyways it is not a problem.

Reply:
Do you need anything else to be modified?